### PR TITLE
updates roadmap: short-term, long term and crazy ideas about Spoon

### PR DESCRIPTION
**Pull request by monperrus**
+ update of the top-level files

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
+    * support for analyzing bound vs unbound type references (`List<T>` vs `List<String>`) 
+* Transformations:
+    * Keep original when printing after transformation
+        * inline comments
+    * indentation and formatting   
+    * sniper mode (only rewrites the changed nodes)
+    * Transactional transformations (rollback if transfo fails)
+    * adds generic transformations that facilitate analysis and transformation
+    * everything in a block
+    * unfinalizer
```
Comment (GerardPaligot) [inline ROADMAP.md]: More information about this?

File: ROADMAP.md [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
+
+* Model
+    * support for analyzing bound vs unbound type references (`List<T>` vs `List<String>`) 
+* Transformations:
+    * Keep original when printing after transformation
+        * inline comments
+    * indentation and formatting   
+    * sniper mode (only rewrites the changed nodes)
+    * Transactional transformations (rollback if transfo fails)
+    * adds generic transformations that facilitate analysis and transformation
```
Comment (GerardPaligot) [inline ROADMAP.md]: what are generic transformations?

File: ROADMAP.md [hunk 3]
```diff
@@ -1,32 +1,38 @@
...
+
+version of 03/09/2015
+
+
+* Model
+    * support for analyzing bound vs unbound type references (`List<T>` vs `List<String>`) 
+* Transformations:
+    * Keep original when printing after transformation
+        * inline comments
+    * indentation and formatting
```
Comment (GerardPaligot) [inline ROADMAP.md]: This point isn't in the previous point "Keep original when printing after transformation"?

File: ROADMAP.md [hunk 4]
```diff
@@ -1,32 +1,38 @@
...
+    * sniper mode (only rewrites the changed nodes)
+    * Transactional transformations (rollback if transfo fails)
+    * adds generic transformations that facilitate analysis and transformation
+    * everything in a block
+    * unfinalizer 
+* Processor orchestration 
+    * specify the current strategy
+        * what if multiple processors on the same node
+        * what if new nodes added?
+    * improved version, see 'Source model analysis using the JJTraveler visitor combinator framework'
```
Comment (GerardPaligot) [inline ROADMAP.md]: A link to 'Source model analysis using the JJTraveler visitor combinator framework'?

**Comment by monperrus** [comment]
done

**Comment by GerardPaligot** [comment]
It's okay for me but ... you can't merge and me neither. :)

**Pull request merged**